reactor-core/src/test/java/reactor/core/ExceptionsTest.java

```diff
@@ -156,9 +156,64 @@ public void propagateDoesntWrapRuntimeException() {
 
 	//TODO test terminate
 
+	@Test
+	void isFatalNotJvmFatalBubbling() {
```

Comment:
WDYT about `bubblingExceptionFatalButNotJvmFatal`? I find the current name really difficult to decipher without reading the test code.

Reply:
good suggestion, done